addon/components/paper-sidenav.js

```diff
@@ -1,75 +1,85 @@
 import Ember from 'ember';
-import PaperNavContainer from './paper-nav-container';
+const { $, inject, run } = Ember;
 
 export default Ember.Component.extend({
-  constants: Ember.inject.service(),
   tagName: 'md-sidenav',
 
-  'locked-open': 'gt-sm',
+  constants: inject.service(),
+  paperSidenav: inject.service('paper-sidenav'),
+
+  setupToggleListener() {
+    this.get('paperSidenav').on('toggle', this, 'onToggle');
+  },
+
+  onToggle(id) {
+    if (id === this.elementId && !this.get('isLockedOpen')) {
+      this.toggleProperty('closed');
+    }
+  },
+
+  /* Defaults */
+  lockedOpen: 'gt-sm',
   closed: true,
   closeOnClick: true,
 
-  navContainer: Ember.computed(function() {
-    return this.nearestOfType(PaperNavContainer);
-  }),
-
+  /* Bindings */
   attributeBindings: ['tabindex'],
-  classNameBindings: ['isLockedOpen:md-locked-open', 'closed:md-closed'],
+  classNameBindings: [
+    'isLockedOpen:md-locked-open',
+    'closed:md-closed'],
   tabindex: -1,
 
-  _init: Ember.on('init', function() {
+  init() {
+    this._super(...arguments);
     let _self = this;
 
-    if (this.get('navContainer')) {
-      this.get('navContainer').set('sideBar', this);
-    }
-
     this.matchMedia();
     this.set('__resizeWindow', function() {
       _self.matchMedia();
     });
-  }),
+
+    this.setupToggleListener();
+  },
 
   _observeClosedState: Ember.observer('closed', function() {
```

Comment:
@miguelcobain I'm using the Evented mixin so that multiple side navs can co-exist and subscribe to toggles to see if they are the one that is targeted. Is there a better way?